pep-0723.rst

```diff
@@ -180,14 +198,15 @@ raised in the Rust pre-RFC.
 Reference Implementation
 ========================
 
-This regular expression may be used to parse the metadata:
+This is the canonical regular expression that may be used to parse the
+metadata:
 
 .. code:: text
 
-   (?ms)^__pyproject__ *= *"""\\?$(.+?)^"""$
+    (?ms)^__pyproject__ *= *"""\\?$(.+?)^"""$
 
 In circumstances where there is a discrepancy between the regular expression
-and the text specification, the text specification takes precedence.
+and the text specification, the regular expression takes precedence.
```

Comment:
In this case, the regular expression should be in *Specification*, as it now serves as the normative specification for parsing.

Reply:
Done!